src/Core/tests/DeviceTests/Handlers/Label/LabelHandlerTests.Android.cs

```diff
@@ -36,6 +36,31 @@ public async Task FontFamilyInitializesCorrectly(string family)
 				Assert.NotEqual(fontManager.DefaultTypeface, nativeLabel.Typeface);
 		}
 
+		[Fact]
+		public async Task PaddingInitializesCorrectly()
+		{
+			var label = new LabelStub()
+			{
+				Text = "Test",
+				Padding = new Thickness(5, 10, 15, 20)
+			};
+
+			var handler = await CreateHandlerAsync(label);
+			var (left, top, right, bottom) = GetNativePadding((TextView)handler.NativeView);
+
+			var context = handler.View.Context;
+
+			var expectedLeft = context.ToPixels(5);
+			var expectedTop = context.ToPixels(10);
+			var expectedRight = context.ToPixels(15);
+			var expectedBottom = context.ToPixels(20);
+
+			Assert.Equal(expectedLeft, left);
+			Assert.Equal(expectedTop, top);
+			Assert.Equal(expectedRight, right);
+			Assert.Equal(expectedBottom, bottom);
+		}
+
```

Comment:
Adds an on-device test to verify that the property is being set on the native Android control. This test will be run on an Android device as part of the CI process.